### PR TITLE
Docs: translate docs/user-guide/rule-deprecation.md

### PR DESCRIPTION
**Pull request by Davont**

### CODE REVIEW

File: docs/user-guide/rule-deprecation.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 
-The ESLint team is committed to making upgrading as easy and painless as possible. To that end, the team has agreed upon the following set of guidelines for deprecating rules in the future. The goal of these guidelines is to allow for improvements and changes to be made without breaking existing configurations.
+ESLint 团队致力于使升级尽可能的简单和无痛。为此，该团队已经就以下一套关于未来废止规则的准则达成了一致。这些准则的目标是允许在不破坏现有配置的情况下进行改进和改变。
 
-* Rules will never be removed from ESLint.
-* Rules will be deprecated as needed, and marked as such in all documentation.
-* After a rule has been deprecated, the team will no longer do any work on it. This includes bug fixes, enhancements, and updates to the rule's documentation. Issues and pull requests related to deprecated rule will not be accepted and will be closed.
+* 规则永远不会从 ESLint 中删除。
+* 规则将根据需要被弃用，并在所有文档中标明。
+* 在一条规则被废弃后，团队将不再对其进行任何工作。这包括错误修复、改进和规则文档的更新。与被废弃的规则有关的问题和拉动请求将不被接受，并且将会被关闭。
```
Comment (QC-L) [inline docs/user-guide/rule-deprecation.md:17]:
```suggestion
* 在一条规则被废弃后，团队将不再对其进行任何工作。这包括错误修复、改进和规则文档的更新。与被废弃的规则有关的问题和 PR 将不被接受，同时将会被关闭。
```

File: docs/user-guide/rule-deprecation.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
-Since deprecated rules will never be removed, you can continue to use them indefinitely if they are working for you. However, keep in mind that deprecated rules will effectively be unmaintained.
+由于废弃的规则永远不会被删除，如果它们对你有用，你可以继续无限期地使用它们。然而，请记住，被废弃的规则实际上将不被维护。
```
Comment (QC-L) [inline docs/user-guide/rule-deprecation.md:19]:
```suggestion
由于废弃的规则永远不会被删除，如果它们对你有用，你可以继续无限期地使用它们。然而，请记住，被废弃的规则实际上将不再维护。
```

File: docs/user-guide/rule-deprecation.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 
-We hope that by following these guidelines we will be able to continue improving and working to make ESLint the best tool it can be while causing as little disruption to our users as possible during the process.
+我们希望通过遵循这些准则，我们能够继续改进并努力使 ESLint 成为最好的工具，同时在这个过程中对我们的用户造成尽可能少的干扰。
```
Comment (QC-L) [inline docs/user-guide/rule-deprecation.md:21]:
```suggestion
我们希望通过遵循这些准则，能够持续改进并努力使 ESLint 成为最好的工具，同时在这个过程中对我们的用户造成尽可能少的干扰。
```

File: docs/user-guide/rule-deprecation.md (hunk 1 above)
Comment (Davont) [reply docs/user-guide/rule-deprecation.md]: > 

谢谢纠正 @QC-L

**Comment by Davont** [comment]
谢谢纠正 @QC-L

**Review by QC-L — APPROVED** [review]

**Pull request merged**